pyinst.py

```diff
@@ -65,7 +65,7 @@ def exe(onedir):
     name = '_'.join(filter(None, (
         'yt-dlp',
         {'win32': '', 'darwin': 'macos'}.get(OS_NAME, OS_NAME),
-        ARCH == '32' and 'x86'
+        '' if MACHINE in ('x86_64', 'amd64') else 'x86' if ('i' in MACHINE and '86' in MACHINE) else MACHINE
```

Comment:
I think it makes sense to put this in a function/global var so that it can be reused below

Reply:
I agree, if you allow I'll prepare something tomorow